_posts/2020-04-17-gh-actions-lessons-learned.md

```diff
@@ -0,0 +1,34 @@
+# GitHub Actions - Lessons Learned
+
+GitHub recently got into the already-crowded CI/CD space with GitHub Actions. For enterprises, you may not want to shake up what already works, but for Open Source Projects, GHA provides a surprising amount of features and an even more enticing price tag: free.
+
+We recently migrated a portion of our builds for pmdarima to GitHub Actions. We're happy with the overall results, but, as early adopters, found some common mistakes that we hope to address here.
```

Comment:
Only one nitpick, can you link to [`pmdarima`](https://github.com/alkaline-ml/pmdarima)

Reply:
Fixed!